conftest.py

```diff
@@ -29,37 +29,46 @@
 import sys
 import warnings
 
+import _pytest.warnings
 import pytest
 
 from stdio_mgr import stdio_mgr
 
 
-def pytest_addoption(parser):
-    """Add custom CLI options to pytest."""
-    parser.addoption(
-        "--skipwarnings",
-        action="store_true",
-        help=("Skip tests that intentionally raise warnings"),
-    )
+@pytest.fixture(scope="session")
```

Comment:
It looks like this fixture is no longer used; if so, can just delete?

Reply:
I dont want to loose it ;-)

I re-integrated it.

Reply:
&lt;nod&gt;, wondered if that were the case. The double-checking assertions aren't a bad thing, I don't think.